examples/flutter_firebase_login/lib/login/bloc/login_event.dart

```diff
@@ -32,11 +32,12 @@ class PasswordChanged extends LoginEvent {
   String toString() => 'PasswordChanged { password: $password }';
 }
 
-class Submitted extends LoginEvent {
+//Class is not being used anywhere in this project; previously 'Submitted'
+class LoginSubmitted extends LoginEvent {
   final String email;
   final String password;
 
-  const Submitted({
+  const LoginSubmitted({
     @required this.email,
     @required this.password,
   });
```

Comment:
if it's not being used, we should just remove it 👍

Reply:
I noticed yesterday. I was about to open an issue or comment about this😅